backbone.js

```diff
@@ -70,6 +70,23 @@
 
   // Proxy Underscore methods to a Backbone class' prototype using a
   // particular attribute as the data argument
+  var modelMatcher = function(attrs) {
+    var matches = _.matches(attrs);
+    return function(model) {
+      return matches(model.attributes);
+    };
+  };
+  var modelProperty = function(key) {
+    return function(model) {
+      return model == null ? void 0 : model.get(key);
+    };
+  };
+  var cb = function(iteratee) {
+    if (_.isFunction(iteratee)) return iteratee;
+    if (_.isObject(iteratee) && !(iteratee instanceof Backbone.Model)) return modelMatcher(iteratee);
```

Comment:
Ah, gotcha. In that case, Collections have `#_isModel` to check for `instanceof` that can be overridden to support polymorphic models.